core/templates/vector.h

```diff
@@ -104,7 +104,9 @@ class Vector {
 	Size rfind(const T &p_val, Size p_from = -1) const { return _cowdata.rfind(p_val, p_from); }
 	Size count(const T &p_val) const { return _cowdata.count(p_val); }
 
-	void append_array(const Vector<T> &p_other);
+	// Must take a copy instead of a reference.
+	// See https://github.com/godotengine/godot/issues/31736.
```

Comment:
Nitpick, but we often reference GitHub issues like this in code comments:
```suggestion
	// Must take a copy instead of a reference (see GH-31736).
```

Reply:
Good to know!